README.md

```diff
@@ -63,9 +63,27 @@ puzzle --mode gemini view -i path/to/gemini_database.db
 Puzzle uses the ped file to show more information in family view and in variant calls:
 
 ```
-puzzle view -i tests/fixtures/hapmap.vcf --family_file/-f tests/fixtures/hapmap.vcf
+puzzle view -i tests/fixtures/hapmap.vcf --family_file/-f tests/fixtures/hapmap.ped
 ```
 
+## Establish a persistent local database
+
+Puzzle can establish a persistent local database, so you can keep work between sessions.
+This enables other key features, such as gene lists for filtering and variant triage.
+First, you need to initialize a local database. Then one or more cases can be loaded and 
+are then ready for viewing as soon as the web server is started.
+
+```
+puzzle init --root tests/fixtures
```

Comment:
Why not establish a persistent database **within** GEMINI's instead of a local one? That is, puzzle particularities prefixed in tables such as `puzzle_pheno`, `puzzle_*` or similar in an already present (GEMINI) database that contains many other useful and easily accessible info?

@robinandeer @moonso @guillermo-carrasco